crates/bevy_gltf/src/loader.rs

```diff
@@ -627,19 +627,35 @@ async fn load_texture<'a>(
     Ok((texture, texture_label(&gltf_texture)))
 }
 
+/// Converts [`TextureTransform`] to [`Mat3`].
+fn texture_transform_mat3(texture_transform: TextureTransform) -> Mat3 {
+    Mat3::from_scale_angle_translation(
+        texture_transform.scale().into(),
+        -texture_transform.rotation(),
+        texture_transform.offset().into(),
+    )
+}
+
 /// Loads a glTF material as a bevy [`StandardMaterial`] and returns it.
 fn load_material(material: &Material, load_context: &mut LoadContext) -> Handle<StandardMaterial> {
     let material_label = material_label(material);
 
     let pbr = material.pbr_metallic_roughness();
 
     let color = pbr.base_color_factor();
-    let base_color_texture = pbr.base_color_texture().map(|info| {
-        // TODO: handle info.tex_coord() (the *set* index for the right texcoords)
-        let label = texture_label(&info.texture());
-        let path = AssetPath::new_ref(load_context.path(), Some(&label));
-        load_context.get_handle(path)
-    });
+    let (base_color_texture, uv_transform) = pbr
+        .base_color_texture()
+        .map(|info| {
+            // TODO: handle info.tex_coord() (the *set* index for the right texcoords)
+            let label = texture_label(&info.texture());
+            let path = AssetPath::new_ref(load_context.path(), Some(&label));
+            (
+                load_context.get_handle(path),
+                info.texture_transform().map(texture_transform_mat3),
+            )
+        })
+        .unzip();
+    let uv_transform = uv_transform.flatten().unwrap_or_default();
```

Comment:
Nit: For code quality, I would prefer if the `base_color_texture` and `uv_transform` definitions were split. the `map(…).unzip()` is fairly hard to parse.